src/index.ts

```diff
@@ -88,7 +88,11 @@ export async function performUpdates<
           _writeProjectManifest: writeProjectManifest,
         }
         const actual = (await fileExists(resolvedPath)) ? await formatPlugin.read(formatHandlerOptions) : null
-        const expected = await formatPlugin.update(clone(actual), updateFile as any, formatHandlerOptions)
+        const expected = await formatPlugin.update(
+          formatPlugin.clone ? formatPlugin.clone(actual, formatHandlerOptions) : clone(actual),
```

Comment:
why are `formatHandlerOptions` useful for the clone function?

Reply:
I don't have a good reason, other than consistency with the other methods. I'm open to removing it.

Reply:
I've removed the `formatHandlerOptions` parameter, as doing so simplifies the code, and I can't think of a compelling reason to keep it.